src/main/java/dk/bec/polonez/reservationsystem/dto/DiscountDto.java

```diff
@@ -0,0 +1,18 @@
+package dk.bec.polonez.reservationsystem.dto;
+
+import lombok.Data;
+
+@Data
+public class DiscountDto {
+
+    private String name;
+
+    private String code;
+
+    private int value;
```

Comment:
**doubt** Is this ```value``` a percentage (like discount = 6%) or some absolute value (like discount = 50 PLN)?
If it's a percentage, then you can consider making a name more precise and readable: ```valueInPercentage```.
If it's an absolute value, you can consider adding currency (like PLN, DKK etc.):

```
public class DiscountDto {

  private AmountDto amount;
}

public class AmountDto {

  private int value;

  private Currency currency;
}
```